doc/base/classes.xml

```diff
@@ -257,13 +257,14 @@
 		<method name="lerp"  >
 			<return type="float">
 			</return>
-			<argument index="0" name="a" type="float">
+			<argument index="0" name="from" type="float">
 			</argument>
-			<argument index="1" name="b" type="float">
+			<argument index="1" name="to" type="float">
 			</argument>
-			<argument index="2" name="c" type="float">
+			<argument index="2" name="percent" type="float">
```

Comment:
"percent" suggests it should be in the 0-100 range.
I think 0.0-1.0 range would be better described as "weight" or "position".